mapbuilder/handlers/jinja.py

```diff
@@ -107,7 +111,7 @@ def to_text(geometry, label: str):
         return ""
 
     labeltext, _, _ = label.partition("#")
-    return f"TEXT:{coord2es(point.coords[0])}:{labeltext}"
+    return f"TEXT:{coord2es(point.coords[0])}:{labeltext}\n"
```

Comment:
Same here. Newlines should be handled by Jinja templates.

Reply:
Okay, I will try to replicate the issues with newlines one more time and will send a result.